config/default/kustomization.yaml

```diff
@@ -1,8 +1,52 @@
 namespace: cabpt-system
 
+namePrefix: cabpt-
```

Comment:
All config/ changes are in support of [this](https://cluster-api.sigs.k8s.io/developer/providers/v1alpha3-to-v1alpha4.html#required-kustomize-changes-to-have-a-single-manager-watching-all-namespaces-and-answer-to-webhook-calls) section of the migration guide.